api/v1beta1/ovndbcluster_types.go

```diff
@@ -34,6 +34,10 @@ const (
 	// DNSSuffix : hardcoded value on how DNSCore domain is configured
 	DNSSuffix = "cluster.local"
 	// TODO: retrieve it from environment
+	// NBDBDnsName - Northbound DNS object name
+	NBDBDnsName = "ovnnb-cluster"
```

Comment:
DNS: https://google.github.io/styleguide/go/decisions#initialisms should retain the original upper case for abbreviations (unless it's unexported and at the start of the variable name, so e.g. dnsList is fine for a variable).